pandas/io/formats/style.py

```diff
@@ -713,7 +741,7 @@ def to_latex(
         >>> df = pd.DataFrame([[1]])
         >>> df.style.set_properties(
         ...     **{"font-weight": "bold /* --dwrap */", "Huge": "--latex--rwrap"}
-        ... ).to_latex(convert_css=True)
+        ... ).to_latex(convert_css=True)  # doctest: +SKIP
```

Comment:
same here: can we doctest the string output? Alternatively if the string output format is not as pretty printed as it is here in the docs then I agree with you that skip is the best option.

Reply:
So, the output string is not printing well:
![image](https://user-images.githubusercontent.com/11354911/127572544-e8de58ff-9cb8-4738-a76a-97a810ce9a04.png)

Reply:
agreed, that is not as helpful to users. thanks for looking.